.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+.hintrc
+.stylelintrc.json
```

Comment:
You should not add the linters files in `.gitignore`, please remove it to have better practices 😉 

- [ ] Remove linters file from `.gitignore`